applications/asset_tracker_v2/doc/location_module.rst

```diff
@@ -0,0 +1,117 @@
+.. _asset_tracker_v2_location_module:
+
+Location module
+###############
+
+.. contents::
+   :local:
+   :depth: 2
+
+The location module controls the positioning functionality of the nRF9160 including GNSS and cellular positioning.
+It can be used to retrieve the location of the device in the form of events containing a position, velocity and time (PVT) structure.
+
+
+Features
+********
+
+This section documents the various features implemented by the module.
+
+Location control
+================
+
+The module uses the :ref:`lib_location` to communicate with the nRF9160 modem and control its GNSS and LTE neighbor cell measurement functionalities.
+A location request is started when the module receives an ``APP_EVT_DATA_GET`` event and the ``APP_DATA_LOCATION`` type is listed in the event's ``data_list`` member containing the data types that shall be sampled.
+
+The default configuration for the location request is set as follows:
+
+* GNSS set as the first priority method and cellular positioning as the second priority.
+  Used location methods can be configured at build time
+  (:ref:`CONFIG_DATA_SAMPLE_GNSS_DEFAULT <CONFIG_DATA_SAMPLE_GNSS_DEFAULT>` and
+  :ref:`CONFIG_DATA_SAMPLE_NEIGHBOR_CELLS_DEFAULT <CONFIG_DATA_SAMPLE_NEIGHBOR_CELLS_DEFAULT>`)
+  or to the dynamically updated value set by the cloud side and stored to flash.
+* Timeout set to either the value configured at build time
+  (:ref:`CONFIG_DATA_LOCATION_TIMEOUT_SECONDS <CONFIG_DATA_LOCATION_TIMEOUT_SECONDS>`)
+  or to the dynamically updated value set by the cloud side and stored to flash.
```

Comment:
**suggestion:** consider clarifying "the dynamically updated value".

Reply:
I've modified this to be aligned with the previous bullet point:
`Timeout which can be configured as part of :ref:`Real-time configurations <real_time_configs>``